updates RNSDK to 4.11.0

cedancp-cs opened the pull request.

cedancp-cs committed:
updates RNSDK to 4.11.0

AldinaSculac commented:
Do you think we should `add` an example for a new way of implementing webviews? Also, I think you need to run `pod install`

big-toni reviewed (approved):
LGTM

big-toni commented:
> @cedancp-cs Do you think we should `add` an example for a new way of implementing webviews? Also, I think you need to run `pod install`
> 
> @big-toni Should we add an example for the crash reporter?

Yes there should be an example for crash reporting. Maybe is better to do it in new PR?

cedancp-cs committed:
feat: adds new webview implementation

cedancp-cs committed:
chore: updates pods

cedancp-cs commented:
> @cedancp-cs Do you think we should `add` an example for a new way of implementing webviews? Also, I think you need to run `pod install`
> 
> @big-toni Should we add an example for the crash reporter?

I updated it to include webview implementation and also updated pods.
Also I created a ticket for including crash reporter

AldinaSculac commented on ClassicImplementation/src/modules/Webviews/views/Webviews.tsx:
```diff
@@ -1,21 +1,33 @@
+        {/* Implementation 1: Webview as a child */}
+        <WebView
+          source={{
+            uri: 'test.com',
```
Should we put here some real URL?

cedancp-cs commented:
> LGTM! I'm just not sure in webview example if we should provide a real URL

hmm I don't think it makes too much difference or does it?
